Dnn.AdminExperience/Library/Dnn.PersonaBar.UI/admin/personaBar/css/main.css

```diff
@@ -933,6 +934,14 @@ span.dnnFormError:after {
     fill: #79bfdb;
 }
 
+.personabarnav>li#Edit.locked svg .back {
+    fill: #0a85c3;

```

Comment:
Is this a color we'd want to define a variable/custom property for?

Reply:
Eventually, yes, but for the 9.10.0 release, this can be hard-coded (as is the case with other element's colors, like text).

Reply:
By the way, this matches the DNN brand color used for the UPDATE button, so I guess that is a good argument for this one staying hard-coded.